src/modal/modal.scss

```diff
@@ -137,16 +125,28 @@
     }
   }
   // #endregion Full Page Modal Animations for CSS
+}
 
-  // #region Full Page Modal Animations for React
-  &.iui-modal-animation-enter > .iui-modal-dialog {
+@mixin iui-modal-animation {
+  // Full Page Modal Animations for React
+  &-enter .iui-modal .iui-modal-dialog {
     transform: translateY(100%);
     opacity: 0;
   }
 
-  &.iui-modal-animation-enter-active > .iui-modal-dialog {
+  &-enter-active .iui-modal .iui-modal-dialog {
```

Comment:
Lines 132 and 137, is the `.iui-modal` part needed?  Would this not work?
```scss
&-enter-active .iui-modal-dialog {
```

Reply:
It wouldnt, because this has higher specificity and would override
![image](https://user-images.githubusercontent.com/81580355/160488552-aab15f28-ae9b-4407-ab0e-adc0b308dd3a.png)

Reply:
What Vykintas says is correct.